src/pages/User/UserState.js

```diff
@@ -233,3 +233,8 @@ export default function MapStateReducer(state = initialState, action = {}) {
       return state;
   }
 }
+
+// selectors
+export const selectors = {
+  getToken: (state) => state.UserState.token
+};
```

Comment:
Selectors work ass accessors, they encapsulate the internal store structure so that the component that reads from the store does not have to know about certain implementation details. This facilitates refactorings within the state structure and potentially provides a base to use more sophisticated performance optimization means like [reselect](https://github.com/reduxjs/reselect).

The more I work with redux, the more I am convinced to move more logic to reducers.